### PR TITLE
Buddhism-and-Semiotics-zensite

### PR DESCRIPTION
**Pull request by JJM317**

### CODE REVIEW

File: _content/essays/buddhism-and-semiotics_fabio-rambelli.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - "Fabio Rambelli"
+external_url: "https://www.thezensite.com/ZenEssays/Philosophical/Buddhism_and_Semiotics.html"
+drive_links:
+  - "https://drive.google.com/file/d/1FMTye0ePjhE3KI1cAdxp9wYdOmUHtTWA/view?usp=sharing"
+course: dialogue
+tags:
+  - east-asian
+  - yogacara
+  - language
+publisher: "The Zen Site"
```
Comment (buddhist-uni) [inline _content/essays/buddhism-and-semiotics_fabio-rambelli.md:13]: Essays don't get / need a publisher but do need a year.

But in any case this was originally an _article_ (yes please move there) published in 1995 in The Semiotic Review of Books (`journal: `)

**Review by buddhist-uni — COMMENTED** [review]
I prefer `_last-first.md` for the filenames

File: _content/article/buddhism-and-semiotics_rambelli-fabio.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  - "Fabio Rambelli"
+external_url: "https://www.thezensite.com/ZenEssays/Philosophical/Buddhism_and_Semiotics.html"
+drive_links:
+  - "https://drive.google.com/file/d/1FMTye0ePjhE3KI1cAdxp9wYdOmUHtTWA/view?usp=sharing"
+course: dialogue
+tags:
+  - east-asian
+  - yogacara
+  - language
+j
```
Comment (buddhist-uni) [inline _content/article/buddhism-and-semiotics_rambelli-fabio.md:13]: You tried to update this one on the website, didn't you? 🤭

**Comment by JJM317** [comment]
busted 🫠

**Pull request merged**